cfme/tests/cli/test_evmserverd.py

```diff
@@ -2,6 +2,7 @@
 """This module contains tests that exercise control of evmserverd service."""
 import pytest
 import re
+from utils import version
 from utils.version import current_version
```

Comment:
Instead of 
```python
from utils import version
from utils.version import current_version
```

let's do

```
from utils.version import current_version, pick
```